generator/internal/codegen_utils_test.cc

```diff
@@ -194,6 +196,36 @@ TEST(ProcessCommandLineArgs, EmptyCommitHash) {
       "--cpp_codegen_opt=googleapis_commit_hash=<hash> must be specified.");
 }
 
+TEST(ProcessCommandLineArgs, NoCopyrighYearParameterOrValue) {
+  auto result = ProcessCommandLineArgs(
+      "product_path=google/cloud/pubsub/,googleapis_commit_hash=foo");
+  auto expected_year =
+      absl::FormatTime("%Y", absl::Now(), absl::LocalTimeZone());
```

Comment:
What is the year changed between the call to `ProcessCommandLineArgs()` and the initialization of `expected year`?  (See the comment about `absl::Now()` in `codegen_utils.cc`.)